File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/xrts/XRT0000Q00LayPacsInfoListItemInfo.java
package nta.med.data.model.ihis.xrts;

import java.io.Serializable;

public class XRT0000Q00LayPacsInfoListItemInfo implements Serializable{

	private String code ;
	private String codeName ;
	private String codeValue ;
	public XRT0000Q00LayPacsInfoListItemInfo(String code, String codeName,
			String codeValue) {
		super();
		this.code = code;
		this.codeName = codeName;
		this.codeValue = codeValue;
	}
	public String getCode() {
		return code;
	}
	public void setCode(String code) {
		this.code = code;
	}
	public String getCodeName() {
		return codeName;
	}
	public void setCodeName(String codeName) {
		this.codeName = codeName;
	}
	public String getCodeValue() {
		return codeValue;
	}
	public void setCodeValue(String codeValue) {
		this.codeValue = codeValue;
	}
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Xrts/XRT0000Q00LayPacsInfoListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Xrts
{
    [Serializable]
	public class XRT0000Q00LayPacsInfoListItemInfo
	{
		private String _code;
		private String _codeName;
		private String _codeValue;

		public String Code
		{
			get { return this._code; }
			set { this._code = value; }
		}

		public String CodeName
		{
			get { return this._codeName; }
			set { this._codeName = value; }
		}

		public String CodeValue
		{
			get { return this._codeValue; }
			set { this._codeValue = value; }
		}

		public XRT0000Q00LayPacsInfoListItemInfo() { }

		public XRT0000Q00LayPacsInfoListItemInfo(String code, String codeName, String codeValue)
		{
			this._code = code;
			this._codeName = codeName;
			this._codeValue = codeValue;
		}

	}
}